tests/test_playbooks/config_group.yml

```diff
@@ -0,0 +1,48 @@
+---
+#
+# Disclaimer: You have to import puppetlabs-ntp module in your foreman instance
+# before running config_group tests
+#
+- hosts: tests
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include: tasks/config_group.yml
+      vars:
+        config_group_state: present
+        expected_change: true
+    - include: tasks/config_group.yml
+      vars:
+        config_group_state: present
+        config_group_puppetclasses: []
+        expected_change: false
+    - include: tasks/config_group.yml
+      vars:
+        config_group_puppetclasses: ['ntp', 'ntp::config']
```

Comment:
does that mean we need to import the `ntp` puppet class before this playbook can run?

Reply:
(or be recorded, to be precise)

Reply:
Yes, as stated in the comment at the top of the file.

Reply:
Right. Comments. Sorry :)